src/lib/assetBridger/erc20Bridger.ts

```diff
@@ -535,15 +535,33 @@ export class Erc20Bridger extends AssetBridger<
       )
       const iGatewayRouter = L1GatewayRouter__factory.createInterface()
 
+      const functionData =
+        defaultedParams.excessFeeRefundAddress !== defaultedParams.from
+          ? iGatewayRouter.encodeFunctionData('outboundTransferCustomRefund', [
+              erc20L1Address,
+              defaultedParams.excessFeeRefundAddress,
+              destinationAddress,
+              amount,
+              depositParams.gasLimit,
+              depositParams.maxFeePerGas,
+              innerData,
+            ])
+          : iGatewayRouter.encodeFunctionData('outboundTransfer', [
+              erc20L1Address,
+              destinationAddress,
+              amount,
+              depositParams.gasLimit,
+              depositParams.maxFeePerGas,
+              innerData,
+            ])
+
+      // Tested to send extra ETH
+      // depositParams.maxSubmissionCost = depositParams.maxSubmissionCost.add(
+      //   parseUnits('0.1', 18)
+      // )
```

Comment:
you need to put this before L532 when `innerData` is calculated and used in `functionData`